app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,101 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+$gray-light : #777;
+$gray :#555;
+$dark-gray : #222;
+
+body {
+    padding-top: 60px;
+}
+
+section{
+    overflow: auto;
+}
+
+textarea{
+    resize: vertical;
+}
+
+.center{
+    text-align: center;
+}
+
+.center h1{
+    margin-bottom: 10px;
+}
+
+
+/* typography */
+h1, h2, h3, h4, h5, h6 {
+    line-height: 1;
+}
+
+h1 {
+    font-size: 3em;
+    letter-spacing: -2px;
+    margin-bottom: 30px;
+    text-align: center;
+}
+
+h2 {
+    font-size: 1.2em;
+    letter-spacing: -1px;
+    margin-bottom: 30px;
+    text-align: center;
+    font-weight: normal;
+    color: #777;
+}
+
+p{
+    font-size: 1.1em; line-height: 1.7em;
+}
+
+/* header */
+#logo {
+    float: left;
+    margin-right: 10px;
+    font-size: 1.7em;
+    color: #fff;
+    text-transform: uppercase;
+    letter-spacing: -1px;
+    padding-top: 9px;
+    font-weight: bold;
+  }
+  #logo:hover {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  img {
+    display: none;
+  }
+
+
+  /* footer */
+footer {
+    margin-top: 45px;
+    padding-top: 5px;
+    border-top: 1px solid #eaeaea;
+    color: $gray-light;
+    a {
+      color:$gray;
+      &:hover{
+        color: $dark-gray ;
+      }
+    }
+    small{
+      float: left;
+    }
+    ul {
+      float: right;
+      list-style: none;
+      li {
+        float: left;
+        margin-left: 15px;
+      }
+    }
+
+  }
```

Comment:
One styling rule @charne has is never use `!important` 😆  ... and she is probably the best person to review styling in our codebases

Reply:
☝🏻 I tend to agree. Its like having a `rescue` or try/catch on the main application thread ... really really bad practise